extension/src/test/runCodeCoverage.ts

```diff
@@ -0,0 +1,23 @@
+import * as path from 'path';
+
+import { runTests } from 'vscode-test';
+
+async function main() {
```

Comment:
this file is redundant. We can use some input argument to distinguish between regular run and code coverage run.

Reply:
I am trying to add an argument to the npm script

Reply:
this file should be deleted.

Reply:
this file should be deleted.

Reply:
Yes, I have added the argument --codecoverage to npm script and removed this file.